tsconfig.json

```diff
@@ -1,35 +1,41 @@
 {
-  "compilerOptions": {
-    "moduleResolution": "node",
-    "module": "es2020",
-    "lib": ["es2021", "DOM"],
-    "target": "es2020",
-    /**
+	"compilerOptions": {
+		"moduleResolution": "node",
+		"module": "es2020",
+		"lib": ["es2021", "DOM"],
+		"target": "es2020",
+		/**
 		svelte-preprocess cannot figure out whether you have a value or a type, so tell TypeScript
 		to enforce using \`import type\` instead of \`import\` for Types.
 		*/
-    "importsNotUsedAsValues": "error",
-    "isolatedModules": true,
-    "resolveJsonModule": true,
-    /**
+		"importsNotUsedAsValues": "error",
+		"isolatedModules": true,
+		"resolveJsonModule": true,
+		/**
 		To have warnings/errors of the Svelte compiler at the correct position,
 		enable source maps by default.
 		*/
-    "sourceMap": true,
-    "esModuleInterop": true,
-    "skipLibCheck": true,
-    "strictNullChecks": true,
-    "noUncheckedIndexedAccess": true,
-    "forceConsistentCasingInFileNames": true,
-    "baseUrl": ".",
-    "allowJs": true,
-    "checkJs": true,
-    "paths": {
-      "$lib": ["src/lib"],
-      "$lib/*": ["src/lib/*"],
-      "$templates": ["src/templates"],
-      "$templates/*": ["src/templates/*"]
-    }
-  },
-  "include": ["src/**/*.d.ts", "src/**/*.js", "src/**/*.ts", "src/**/*.svelte"]
+		"sourceMap": true,
+		"esModuleInterop": true,
+		"skipLibCheck": true,
+		"strictNullChecks": true,
+		"noUncheckedIndexedAccess": true,
+		"forceConsistentCasingInFileNames": true,
+		"baseUrl": ".",
+		"allowJs": true,
+		"checkJs": true,
+		"paths": {
+			"$lib": ["src/lib"],
+			"$lib/*": ["src/lib/*"],
+			"$templates": ["src/templates"],
+			"$templates/*": ["src/templates/*"],
+			"$app/*": ["node_modules/@sveltejs/kit/types"]
```

Comment:
Added `$app/*` because typescript was unable to resolve Svelte's `$app/navigate` module outside of `*.svelte` files (e.g. in [`reload.ts`](https://github.com/guardian/commercial-templates/blob/1ed099339327d933301a772ca7ef9cd8d9a40c08/src/lib/reload.ts#L1))